sources/api/migration/migrations/v1.21.0/build-kubernetes-pause-container/src/main.rs

```diff
@@ -0,0 +1,35 @@
+use migration_helpers::common_migrations::{
+    AddPrefixesMigration, MetadataListReplacement, RemoveMetadataMigration,
+    ReplaceMetadataListsMigration, SettingMetadata,
+};
+use migration_helpers::{migrate, Result};
+use std::process;
+
+fn run() -> Result<()> {
+    migrate(RemoveMetadataMigration(&[SettingMetadata {
```

Comment:
I couldn't find any other migrations with multiple independent migrate calls.  Presumably we get into a bad state if the first succeeds and second fails.

Reply:
SGTM, thanks, and fixed!